tests/tools/importexport/test_reader.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Tests for archive reader."""
+# pylint: disable=pointless-statement,redefined-outer-name
+import pytest
+
+from aiida.common import InvalidOperation
+from aiida.tools.importexport import EXPORT_VERSION, CorruptArchive
+from aiida.tools.importexport.archive import get_reader
+from tests.utils.archives import get_archive_file
+
+
+@pytest.fixture
+def archive_reader():
+    """Return an initiated archive reader."""
+
+    def _func(filename=None):
+        archive_path = get_archive_file(filename or f'export_v{EXPORT_VERSION}_simple.aiida', 'export/migrate')
+        reader_cls = get_reader('zip')
+        return reader_cls(archive_path)
+
+    return _func
+
+
+def test_reader(archive_reader):
+    """Test reader API."""
+
+    with archive_reader() as archive:
+        assert archive.export_version == '0.9'
```

Comment:
looking a the filename of the file, this could be replaced by `EXPORT_VERSION` (otherwise one has to always fix it here).

Reply:
done